contents/dotfiles/Dotfiles.md

```diff
@@ -0,0 +1,233 @@
+<frontmatter>
+  title: Introduction to Dotfiles
+  header: pagetop.md
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 4
+</frontmatter>
+
+<div class="website-content">
+
+{{ booktitle | safe }}
+
+# Introduction to Dotfiles
+
+**Author(s): [Tiu Wee Han](https://github.com/tiuweehan)**<br>
+Reviewers: [James Pang](https://github.com/jamessspanggg), [Liu Yiwen](https://github.com/0blivious)
+
+<box id="article-toc">
+
+* [Introduction to Dotfiles](#introduction-to-dotfiles)
+    * [What are Dotfiles?‎](#what-are-dotfiles)
+    * [Why Dotfiles?](#why-dotfiles)
+        * [Benefit: Save and Reuse Configurations](#benefit-save-and-reuse-configurations)
+        * [Benefit: Personalised Shortcuts and Commands‎](#benefit-personalised-shortcuts-and-commands‎)
+        * [Benefit: Shell Utility and Styling](#benefit-shell-utility-and-styling)‎
+    * [Getting Started](#getting-started)
+        * [Management Strategies‎](#management-strategies)
+    * [Conclusion](#conclusion)
+
+</box>
+
+### What are Dotfiles?
+Dotfiles are plain text configuration files on Unix like systems (e.g. MacOS, Linux, BSD). Dotfiles store settings of almost every application, service and tool running on your system. These files control the behavior of applications from boot to termination and everything in between.
+
+<pic src="dotfiles-logo.png" alt="Dotfiles Logo" width="45%">
+  
+  <sub>_Figure 1. Dotfiles Logo_ [(source)](https://www.twilio.com/blog/using-dotfiles-productivity-bootstrap-systems)</sub>
+
+</pic>
+
+Some common uses of dotfiles include:
+- Scripts that run when a Shell is started: `.profile`, `.bashrc`, `.zshrc`
+- Customising text editors: `.nanorc`, `.vimrc`, `.emacs.d`
+- Storing settings for commonly used tools (e.g. `git`, `ssh`): `.gitconfig`, `.gitignore`, `.ssh/config`
+
+### Why Dotfiles?
+
+#### Benefit 1: Configure Applications
```

Comment:
Phrase all three benefits the same way? This one sounds like an action while the other two don't.